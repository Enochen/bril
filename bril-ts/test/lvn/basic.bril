@main {
  a: int = const 4;
  b: int = const 4;
  c: int = const 4;
  sum1: int = add a b;
  sum2: int = add b c;
  sum3: int = add c a;
  sum1: int = const 4;
  print sum1;
}
