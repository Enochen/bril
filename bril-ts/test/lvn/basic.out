@main {
  a: int = const 4;
  sum1: int = id a;
  print sum1;
}
